docs/user-guide/ze-create-zowe-explorer-cics-ext-profiles.md

```diff
@@ -1,99 +1,77 @@
-# Creating Zowe Explorer CICS Extension profiles
+# Managing Zowe Explorer CICS Extension profiles
 
-You need to have a Zowe Explorer profile to use all functions of the extension.
+To use all functions of the Zowe Explorer CICS extension, you need to have a Zowe Explorer profile.
 
-If you already have a Zowe CICS CLI profile, the CICS tree loads the default profile on startup.  
+If you already have a Zowe CICS CLI profile, the CICS tree in the VS Code **Side Bar** loads the default profile on startup.  
 
-If you do not have an existing Zowe CICS CLI profile, follow these steps to create one:
+If you do not have an existing Zowe CICS CLI profile, refer to the following instructions.
 
 ## Using Zowe team configuration
 
-1. Select the **+** button in the CICS tree.
+1. Expand the **CICS** tree in the **Side Bar**, and click the **+** icon.
 
-2. Select the **Create New CICS profile** option to open the config file.
+2. In the **Quick Pick** menu, select the **Create New CICS profile** option to open the configuration file.
 
-3. Edit the config file to add a CICS profile.
+   The associated **[is there a better way/word to describe this?]** configuration file opens in an **Editor**.
```

Comment:
@adam-wolfe - please respond

Reply:
Maybe we could change this to "The active configuration file opens...", however you could have more than one active config. I'm also not sure how V3 will behave, if you don't have a config in place, I assume that one of the options (or maybe the only option) in the Quick Pick menu would be to "Create a new Team Configuration file".

@zFernand0

Reply:
I agree that we should change it to "The active configuration file".

I believe it will only have the one option to create a new team config file. 😋